src/components/Avatar/Avatar.scss

```
@@ -1,9 +1,10 @@
 @use '../variables';
+@use './variables' as avatarVariables;
```

```suggestion
@use './variables' as avatar-variables;
```

We prefer kebab-case in styles